File: next/security/backend-ms1/src/main/java/ma/zs/stocky/ws/dto/security/UserDto.java
package  ma.zs.stocky.ws.dto.security;

import ma.zs.stocky.zynerator.audit.Log;
import ma.zs.stocky.zynerator.dto.AuditBaseDto;
import com.fasterxml.jackson.annotation.JsonInclude;

import java.util.List;




@JsonInclude(JsonInclude.Include.NON_NULL)
public class UserDto  extends AuditBaseDto {

    private Boolean credentialsNonExpired  ;
    private Boolean enabled  ;
    private String email  ;
    private Boolean accountNonExpired  ;
    private Boolean accountNonLocked  ;
    private String username  ;
    private String password  ;
    private Boolean passwordChanged  ;
    private String lastName  ;
    private String firstName  ;
    private String phone  ;


    private List<RoleUserDto> roleUsers ;


    public UserDto(){
        super();
    }



    @Log
    public Boolean getCredentialsNonExpired(){
        return this.credentialsNonExpired;
    }
    public void setCredentialsNonExpired(Boolean credentialsNonExpired){
        this.credentialsNonExpired = credentialsNonExpired;
    }

    @Log
    public Boolean getEnabled(){
        return this.enabled;
    }
    public void setEnabled(Boolean enabled){
        this.enabled = enabled;
    }

    @Log
    public String getEmail(){
        return this.email;
    }
    public void setEmail(String email){
        this.email = email;
    }

    @Log
    public Boolean getAccountNonExpired(){
        return this.accountNonExpired;
    }
    public void setAccountNonExpired(Boolean accountNonExpired){
        this.accountNonExpired = accountNonExpired;
    }

    @Log
    public Boolean getAccountNonLocked(){
        return this.accountNonLocked;
    }
    public void setAccountNonLocked(Boolean accountNonLocked){
        this.accountNonLocked = accountNonLocked;
    }

    @Log
    public String getUsername(){
        return this.username;
    }
    public void setUsername(String username){
        this.username = username;
    }

    @Log
    public String getPassword(){
        return this.password;
    }
    public void setPassword(String password){
        this.password = password;
    }

    @Log
    public Boolean getPasswordChanged(){
        return this.passwordChanged;
    }
    public void setPasswordChanged(Boolean passwordChanged){
        this.passwordChanged = passwordChanged;
    }

    @Log
    public String getLastName(){
        return this.lastName;
    }
    public void setLastName(String lastName){
        this.lastName = lastName;
    }

    @Log
    public String getFirstName(){
        return this.firstName;
    }
    public void setFirstName(String firstName){
        this.firstName = firstName;
    }

    @Log
    public String getPhone(){
        return this.phone;
    }
    public void setPhone(String phone){
        this.phone = phone;
    }





    public List<RoleUserDto> getRoleUsers(){
        return this.roleUsers;
    }

    public void setRoleUsers(List<RoleUserDto> roleUsers){
        this.roleUsers = roleUsers;
    }



}

File: laravel/v3/backend-ms1/app/Dto/security/UserDto.php
<?php

namespace App\Dto\security;


class UserDto
{

    private $credentialsNonExpired;
    private $enabled;
    private $email;
    private $accountNonExpired;
    private $accountNonLocked;
    private $username;
    private $password;
    private $passwordChanged;
    private $lastName;
    private $firstName;
    private $phone;




    public function getId() {
        return $this->id;
    }
    public function setId($id){
        $this->id = $id;
    }

    public function getCredentialsNonExpired() {
        return $this->credentialsNonExpired;
    }
    public function setCredentialsNonExpired($credentialsNonExpired){
        $this->credentialsNonExpired = $credentialsNonExpired;
    }

    public function getEnabled() {
        return $this->enabled;
    }
    public function setEnabled($enabled){
        $this->enabled = $enabled;
    }

    public function getEmail() {
        return $this->email;
    }
    public function setEmail($email){
        $this->email = $email;
    }

    public function getAccountNonExpired() {
        return $this->accountNonExpired;
    }
    public function setAccountNonExpired($accountNonExpired){
        $this->accountNonExpired = $accountNonExpired;
    }

    public function getAccountNonLocked() {
        return $this->accountNonLocked;
    }
    public function setAccountNonLocked($accountNonLocked){
        $this->accountNonLocked = $accountNonLocked;
    }

    public function getUsername() {
        return $this->username;
    }
    public function setUsername($username){
        $this->username = $username;
    }

    public function getPassword() {
        return $this->password;
    }
    public function setPassword($password){
        $this->password = $password;
    }

    public function getPasswordChanged() {
        return $this->passwordChanged;
    }
    public function setPasswordChanged($passwordChanged){
        $this->passwordChanged = $passwordChanged;
    }

    public function getLastName() {
        return $this->lastName;
    }
    public function setLastName($lastName){
        $this->lastName = $lastName;
    }

    public function getFirstName() {
        return $this->firstName;
    }
    public function setFirstName($firstName){
        $this->firstName = $firstName;
    }

    public function getPhone() {
        return $this->phone;
    }
    public function setPhone($phone){
        $this->phone = $phone;
    }

    public function getRoleUsers() {
        return $this->roleUsers;
    }
    public function setRoleUsers($roleUsers){
        $this->roleUsers = $roleUsers;
    }

    public function getModelPermissionUsers() {
        return $this->modelPermissionUsers;
    }
    public function setModelPermissionUsers($modelPermissionUsers){
        $this->modelPermissionUsers = $modelPermissionUsers;
    }

}


